### ISSUE TITLE
A11y_AzureDataStudio_CheckDependencies_AI4D:Name property does not exist for the expand/collapse button.

### ISSUE THREAD

**Issue by vinodbk197** [microsoft/azuredatastudio#7624]
"[Check out Accessibility Insights!](https://accessibilityinsights.io) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHubTags:**#A11y_AzureDataStudio;#A11yMAS; #A11yTCS;#SQL Azure Data Studio;#WCAG1.3.1;#WCAG4.1.2;#DesktopApp;#Win32;#AI4D;#NVDA;#JAWS;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 32df727
Date: 2019-10-08T19:18:43.743Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

**Additional Details:**
Tool Name : AI4D

**Repro Steps:**
1.	Install /Launch the application Azure data studio.
2.	Navigate to connections and click on more action button.
3.	Select “Deploy SQL Server big data cluster” from more action drop down.
4.	Click on “Deploy SQL Server big data cluster”.
5.	Check the check box I accept in the “select the deployment option” screen and click on 
        select button.
6.	Select any block and click on next button and again click on next button.
7.	Enter password and confirm password and click on next button and again click on next 
        button and click on script to Notebook ->Navigate till Check dependencies.
8.	Open AI4D and check the name property for the button(^).

**Actual:**
For expand/collapse (^) button, Name property does not exist.

**Expected:**
For expand/collapse (^) button, Name property should provide and screen reader should announces the same.

**User Impact:**
Screen reader user does not understand regarding the name of the control.

**Attachment for Reference:**
[7624_A11y_AzureDataStudio_CheckDependencies_AI4D](https://github.com/microsoft/azuredatastudio/files/3712068/7624_A11y_AzureDataStudio_CheckDependencies_AI4D.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by alanrenmsft** [comment]
@corivera and @chlafreniere could you please take a look?
<img width="982" alt="Screen Shot 2019-11-22 at 11 10 59 AM" src="https://user-images.githubusercontent.com/13777222/69453460-cd1fa200-0d18-11ea-82e4-315d5891c95b.png">

**Comment by chlafreniere** [comment]
Should be closed by #8607

**Comment by vinodbk197** [comment]
Verified the issue on:Azure Data Studio
Version: 1.15.0-insider (user setup)
![image](https://user-images.githubusercontent.com/55909580/71977463-dae4a480-323e-11ea-9a66-206c42630d3e.png)

The issue is resolved ,Hence closing this bug.

**Comment by fsteffi** [comment]
#Azure Data Studio - New Features_win32;

**Comment by Yash14j** [comment]
#A11ysev2;#Closed;
